src/algebra/lie_algebra.lean

```diff
@@ -305,18 +305,18 @@ instance lie_submodule_coe_submodule [lie_module R L M] :
   has_coe (lie_submodule R L M) (submodule R M) := ⟨lie_submodule.to_submodule⟩
 
 instance lie_submodule_has_mem [lie_module R L M] :
-  has_mem M (lie_submodule R L M) := ⟨λ x M', x ∈ (M' : set M)⟩
+  has_mem M (lie_submodule R L M) := ⟨λ x N, x ∈ (N : set M)⟩
 
-instance lie_submodule_lie_module [α : lie_module R L M] [M' : lie_submodule R L M] :
-  lie_module R L M' :=
+instance lie_submodule_lie_module [α : lie_module R L M] [N : lie_submodule R L M] :
```

Comment:
Ooh, the `lie_submodule` shouldn't be an instance variable. Use `()` instead of `[]`.